index.html

```diff
@@ -10,7 +10,7 @@
   </head>
   <body>
     <div id="app">
-      <svg viewBox="0 0 420 420" width="100" height="100" xmlns="http://www.w3.org/2000/svg" style="background: #F5F5E9;">
+      <svg viewBox="0 0 420 420" width="100" height="100" xmlns="http://www.w3.org/2000/svg" style="background: #F5F5E9;" id="logo">
```

Comment:
```suggestion
      <svg viewBox="0 0 420 420" width="100" height="100" xmlns="http://www.w3.org/2000/svg" style="background: #F5F5E9;" class="logo">
```
use a class instead of ID for styling. ID's are too specific, only use them in your JS, or as a page anchor.